.github/workflows/run-metrics.yml

```
@@ -0,0 +1,33 @@
+  push:
+    paths: ['cultivation/data/processed/**']
+jobs:
+  metrics:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: sudo apt-get update && sudo apt-get install -y jq bc
+      - run: pip install -r requirements.txt
+      - run: python cultivation/scripts/running/aggregate_weekly_runs.py
+      - id: kpi
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)